packages/api/test.tap

```diff
@@ -1,32 +1,2 @@
-1..2
+1..75
 Example app listening at http://localhost:3000
-ok 1 Repository performTicketIfAllowed deletes test if permitted
-ok 2 Repository performTicketIfAllowed does not delete test if not permitted
-# tests 2
-# pass 2
```

Comment:
If you don't mind let's delete the `.tap` file. Would love to know how these keep sneaking in, it might be worth double checking whether this file is staged for commit next time you're pushing anything, perhaps we've screwed up the ignore rules.